client/style.css

```diff
@@ -416,17 +416,25 @@ h1{
   float: right;
 }
 
-.add-question-form label {
+.add-question-form label, .add-answer-form label {
   width: 10%;
   text-align: right;
   display: inline-block;
   padding-right: 1%;
 }
 
-.add-question-form input, .add-question-form textarea{
+.add-question-form input, .add-question-form textarea, .add-answer-form input, .add-answer-form textarea{
   width: 50%
 }
 
+.qa-add-answer {
```

Comment:
I see spaces and no spaces between class/id tags and the braces. I have been using no spaces but we should arrive at a rule and go with that.

Reply:
Sure, we can agree to a rule tomorrow.